pkg/host/internal/sriov/sriov.go

```diff
@@ -115,12 +115,15 @@ func (s *sriov) GetVfInfo(pciAddr string, devices []*ghw.PCIDevice) sriovnetwork
 		VfID:       id,
 	}
 
-	if mtu := s.networkHelper.GetNetdevMTU(pciAddr); mtu > 0 {
-		vf.Mtu = mtu
-	}
 	if name := s.networkHelper.TryGetInterfaceName(pciAddr); name != "" {
-		vf.Name = name
-		vf.Mac = s.networkHelper.GetNetDevMac(name)
+		link, err := s.netlinkLib.LinkByName(name)
+		if err != nil {
+			log.Log.Error(err, "getVfInfo(): unable to get VF Link Object", "name", name, "device", pciAddr)
+		} else {
+			vf.Name = name
```

Comment:
Do we want to keep Name set in case of failure?

Reply:
it keeps same logic as before, if TryGetInterfaceName returns an interface, the name is set.

im fine with making this more strict and set if we can get the link device.